### ISSUE TITLE
Non ACTG Nucleotides

### ISSUE THREAD

**Issue by StephanHolgerD** [igvteam/igv-reports#107]
Hello, 
great package thanks for providing this :)

I have a question regarding non-ACTG nucleotides in BAM files. 

I try to visualize a BAM file that contains alignments with nucleotide representations like the follwoing ones:

K	Guanine / Thymine
M	Adenine / Cytosine
S	Guanine / Cytosine
W	Adenine / Thymine

Somehow, igv-reports don’t show anything for positions with those nucleotides, just empty fields:

![image](https://github.com/igvteam/igv-reports/assets/28348674/91c8d734-e2a5-4ad1-83dd-692544c7769a)

Is it possible to visualize such an alignment file properly?

**Comment by jrobinso** [comment]
This is fixed in igv.js, but can't be fixed in igv-reports until igv.js release.  As a workaround or test you can use the snapshot igv.js build in your reports, i.e. change the igv.js script element in the template to

```
    <script src="https://igv.org/web/snapshot/dist/igv.min.js"></script>
```

**Comment by StephanHolgerD** [comment]
Thanks, will test this asap

**Comment by StephanHolgerD** [comment]
[test_bam.txt](https://github.com/igvteam/igv-reports/files/14444647/test_bam.txt)

Hey I testest the new igv.min.js but somehow the nucleotides are still not visible, I uploaded the the html (I just changed the ending to .txt because github does not allow .html uploads here)

**Comment by jrobinso** [comment]
The dev build was probably not ready yet,  it build nightly, sorry I should
have mentioned that.   I see them now with your test.

**Comment by StephanHolgerD** [comment]
No problem, I really appreciate your help!

**Comment by jrobinso** [comment]
This should be fixed now in igv-reports version 1.13.0.   Old reports can be patched by updating the igv.js version to 3.0.2
